gallery/src/demos/demo-hui-glance-card.js

```diff
@@ -1,77 +1,130 @@
+entities:
+  - device_tracker.demo_paulus
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights
+  - lock.kitchen_door
+  - light.ceiling_lights`
   },
   {
-    entities: [
-      'binary_sensor.movement_backyard',
-      'light.bed_light',
-      'binary_sensor.basement_floor_wet',
-      'sensor.outside_temperature',
-      'light.ceiling_lights',
-      'switch.ac',
-      'lock.kitchen_door'
-    ],
-    type: 'glance',
+    heading: 'With title',
+    config: `type: glance
+title: This is glance
+entities:
+  - device_tracker.demo_paulus
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights
+  - lock.kitchen_door
+  - light.ceiling_lights`
   },
   {
-    entities: [
-      'lock.kitchen_door'
-    ],
-    type: 'glance',
+    heading: 'Custom column width',
+    config: `type: glance
+column_width: calc(100% / 7)
+entities:
+  - device_tracker.demo_paulus
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights
+  - lock.kitchen_door
+  - light.ceiling_lights`
+  },
+  {
+    heading: 'No name',
+    config: `type: glance
+show_name: false
+entities:
+  - device_tracker.demo_paulus
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights
+  - lock.kitchen_door
+  - light.ceiling_lights`
+  },
+  {
+    heading: 'No state',
+    config: `type: glance
+show_state: false
+entities:
+  - device_tracker.demo_paulus
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights
+  - lock.kitchen_door
+  - light.ceiling_lights`
+  },
+  {
+    heading: 'No name and no state',
+    config: `type: glance
+show_name: false
+show_state: false
+entities:
+  - device_tracker.demo_paulus
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights
+  - lock.kitchen_door
+  - light.ceiling_lights`
+  },
+  {
+    heading: 'Custom name',
+    config: `type: glance
+entities:
+  - entity: device_tracker.demo_paulus
+    name: ¯\\_(ツ)_/¯
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights
+  - lock.kitchen_door
+  - light.ceiling_lights`
+  },
+  {
+    heading: 'Custom tap action',
+    config: `type: glance
+entities:
+  - entity: lock.kitchen_door
+    tap_action: toggle
+  - entity: light.ceiling_lights
+    tap_action: turn-on
+  - device_tracker.demo_paulus
+  - media_player.living_room
+  - sun.sun
+  - cover.kitchen_window
+  - light.kitchen_lights`
   },
-
 ];
 
 class DemoPicEntity extends PolymerElement {
   static get template() {
     return html`
-      <style>
-        #root {
-        }
-        hui-glance-card {
-          width: 400px;
-          display: inline-block;
-          margin-left: 20px;
-          margin-top: 20px;
-        }
-      </style>
-      <div id='root'>
-      </div>
+      <demo-card type="hui-glance-card" configs="[[_configs]]"></demo-card>
```

Comment:
Let's call the component demo-cards as it renders multiple. Inside demo-cards we can then have a component demo-card ?

Reply:
So 1 is repsonible for layouting multiple demo-cards, demo-card is responsible for rendering 1 card

That way we can start adding toggles to show config on demand etc